src/components/bar.jsx

```diff
@@ -5,11 +5,19 @@ import {
   Component,
 } from 'react';
 
+import BarRectangle from './bar_rectangle';
+
 import { series } from '../utils/series';
 
 export default class Bar extends Component {
   constructor(props) {
     super(props);
+    this.barRadius = {
```

Comment:
Should these values (or a single value, copied, since they are the same) e passed in as a prop instead of being hard-coded to a single value?